internal/endpoints/common.go

```diff
@@ -35,12 +37,44 @@ func Login(w http.ResponseWriter, r *http.Request) {
 	jwtRef := session.NewJWT()
 	err := jwtRef.Build(userSession)
 	if err != nil {
-		log.Printf("Could not build JWT: %+v", err)
-		w.WriteHeader(http.StatusInternalServerError)
+		log.Printf("[ERROR] Could not build JWT: %+v", err)
```

Comment:
We could switch to [github.com/google/logger](https://github.com/google/logger) to get more sophistication in logging.

Reply:
Ok will take a look. I'll try adding it to the next PR